package.json

```diff
@@ -262,7 +262,7 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "^7.17.7",
-    "@department-of-veterans-affairs/component-library": "^11.2.1",
+    "@department-of-veterans-affairs/component-library": "^11.2.2",
```

Comment:
This is the [release with the published fix](https://github.com/department-of-veterans-affairs/component-library/releases/tag/v11.2.2).